Code/GraphMol/testChirality.cpp

```diff
@@ -631,6 +632,274 @@ void testChiralityCleanup() {
   TEST_ASSERT(mol->getAtomWithIdx(1)->getChiralTag() == Atom::CHI_UNSPECIFIED);
   delete mol;
 
+  // The remaining examples are for github #1614 :
+  // AssignStereochemistry incorrectly removing CIS/TRANS bond stereo
+
+  // cleanIt=true, force=true should NOT remove this manual cis/trans stereo
+  // assignment
+  smi = "CC=CC(Cl)C";
```

Comment:
Just manually reviewed all the SMILES in this test case. All the assertions look correct to me.